### ISSUE TITLE
Initial setup failing with "no value for network in /home/admin/raspiblitz.info"

### ISSUE THREAD

**Issue by raucao** [raspiblitz/raspiblitz#2394]
I followed [the instructions for building the SD card image](https://github.com/rootzoll/raspiblitz#build-the-sd-card-image), and the initial setup script is failing with the following error:

```
FAIL: Something is wrong. There is no value for network in /home/admin/raspiblitz.info.
Should be at least default value. EXIT
```

The config key does not exist in that file, indeed. But there's no info I could find in this repo about what the value should be. I also couldn't find any default being set in the build script.

**Comment by openoms** [comment]
the build_sdcard.sh defaults to the `dev` branch which is behind.

Would suggest to reflash with the 64bit RAspberry OS image and run the build with the options:

`sudo bash build_sdcard.sh false false rootzoll v1.7`

**Comment by raucao** [comment]
I tried that just now, but it's yielding the same result:

```
admin@raspberrypi:~ $ sudo cat /home/pi/.bash_history |grep build
wget https://raw.githubusercontent.com/rootzoll/raspiblitz/v1.7/build_sdcard.sh
sudo bash build_sdcard.sh false false rootzoll v1.7
admin@raspberrypi:~ $ ./10setupBlitz.sh 
checking setup script
network()
chain()
setupStep()
FAIL: Something is wrong. There is no value for network in /home/admin/raspiblitz.info.
Should be at least default value. EXIT
```

**Comment by openoms** [comment]
Not sure what is  happening there.

Flash the Raspberry OS fresh then run in pi user:
```
wget https://raw.githubusercontent.com/rootzoll/raspiblitz/v1.7/build_sdcard.sh
sudo bash build_sdcard.sh false false rootzoll v1.7
```
The setup process and menu will start automatically once you reboot and login with the admin user.

**Comment by raucao** [comment]
I added the grep'ed bash history to demonstrate that this is exactly what I did.

Running the script manually is also just for demonstrating the result. It did start automatically on first admin login, and failed with the same error message.

**Comment by raucao** [comment]
The code here definitely does not write a `network` config to that info file:

https://github.com/rootzoll/raspiblitz/blob/v1.7/build_sdcard.sh#L555-L558

But the same 1.7 branch's setup script also definitely requires it:

https://github.com/rootzoll/raspiblitz/blob/v1.7/home.admin/10setupBlitz.sh#L17-L21

**Comment by openoms** [comment]
the info file is filled by the `_bootstrap.sh` run by `/etc/systemd/system/bootstrap.service`

Check 
`sudo systemctl status bootstrap.service`

Restart:
`sudo systemctl restart bootstrap.service`

Or restart the whole node again.

Would recommend to rebuild if something has gone wrong in the start to have a clean state.

**Comment by raucao** [comment]
Aha, now we're getting closer! The bootstrap script seems to be hanging at `btrfs filesystem show`.

I checked, and the HDD is still unformatted. `/var/log/bootstrap.log` is empty. I verified that running the above command doesn't work (but also doesn't fail, as it just hangs).

The `btrfs filesystem show` commands (multiple processes) are coming from `/home/admin/config.scripts/blitz.datadrive.sh`. I assume they're supposed to gracefully fail instead of hang. I'll check the disk to see if I can find out more...

Update: I found out that the disk throws I/O errors when trying to read from it. I guess it's on its way to nirvana. Not sure there's a good way to account for a faulty disk in the bootstrap script, so I'll close this issue. Maybe it still helps someone with the same problem if they can find this in the future.

@openoms Thanks for your help! Much appreciated.
